components/malloc_size_of/Cargo.toml

```diff
@@ -9,7 +9,9 @@ publish = false
 path = "lib.rs"
 
 [features]
-servo = ["mozjs", "string_cache", "url", "webrender_api", "xml5ever"]
+servo = ["mozjs", "string_cache", "xml5ever"]
+url_traits = ["url"]
+webrender_traits = ["webrender_api"]
```

Comment:
Adding two features that enable one crate each is probably unnecessary. Optional dependencies can already be used as features: `#[cfg(feature = "url")]`, `features = ["url"]`

Reply:
I didn't know you could do that, it certainly makes things a lot cleaner. :) I'll make the changes tomorrow.